eclipse-che/che-dashboard
# Add plugin registry environment variables

Signed-off-by: Oleksii Orel <oorel@redhat.com>



### What does this PR do?
Add plugin registry environment variables if creating devworkspaces from generated resources.

### What issues does this PR fix or reference?
fixed https://github.com/eclipse/che/issues/21054



### How to test this PR?


1. Deploy Eclipse-CHE with devWorkspace enabled.
2. Open the 'Create Workspace' page.
3. Click on one of the samples(anyone: 'Bash', 'Apache Camel K' ...).
4. Click on View/Plugins. The plugin list should be shown.
